rclcpp/executors/multithreaded_executor/multithreaded_executor.cpp

```diff
@@ -101,8 +99,9 @@ class DualThreadedNode : public rclcpp::Node
                                             // So we just put a generic placeholder
                                             // into the binder
                                             // (since we know we need ONE parameter)
-      sub1_opt);                  // This is where we set the callback group.
-                                  // This subscription will run with callback group subscriber1
+      sub1_opt,
+      callback_group_subscriber1_);         // This is where we set the callback group.
+                                            // This subscription will run with callback group subscriber1
```

Comment:
Maybe you have noticed `Lines should be <= 100 characters long` in the CI warning.

Reply:
nice catch 👍 done